dae/dae/variants_loaders/vcf/tests/test_vcf_loader_multi.py

```diff
@@ -0,0 +1,674 @@
+# pylint: disable=W0621,C0114,C0116,W0212,W0613
+import os
+from pathlib import Path
+from typing import Any, Callable, Union
+import pytest
+import numpy as np
+from dae.variants_loaders.vcf.loader import VcfLoader
+from dae.genomic_resources.testing import setup_pedigree, setup_vcf
+from dae.gpf_instance.gpf_instance import GPFInstance
+from dae.testing.acgt_import import acgt_gpf
+from dae.pedigrees.loader import FamiliesLoader
+
+
+@pytest.fixture
+def gpf_instance(
+        tmp_path_factory: pytest.TempPathFactory) -> GPFInstance:
+    root_path = tmp_path_factory.mktemp("instance")
+    gpf_instance = acgt_gpf(root_path)
+    return gpf_instance
+
+
+@pytest.fixture
+def multivcf_split1_vcf(tmp_path_factory: pytest.TempPathFactory) -> Path:
+    root_path = tmp_path_factory.mktemp("mutlivcf_split1")
+    vcf_path = setup_vcf(root_path / "vcf_data" / "in.vcf", """
+    ##fileformat=VCFv4.2
+    ##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+    ##INFO=<ID=EFF,Number=1,Type=String,Description="Effect">
+    ##contig=<ID=1>
+    #CHROM	POS	    ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	f1.mom	f1.dad	f1.p1	f1.s1	f2.mom	f2.dad	f2.p1	f2.s1	f3.mom	f3.dad	f3.p1	f3.s1
+    1	    865582	.	C	T	.	    .	    EFF=SYN	GT	    1/1	    0/0 	0/1 	0/1 	1/1 	0/0 	0/1 	0/1 	1/1 	0/0 	0/1 	0/1
+    1	    865583	.	G	A	.	    .	    EFF=SYN	GT	    0/0	    1/1 	0/1 	0/1 	0/0 	1/1 	1/0 	0/1 	0/0 	1/1 	0/1 	0/1
+    1	    865624	.	G	A	.	    .	    EFF=MIS	GT	    1/0	    0/0 	0/1 	0/0 	1/0 	0/0 	0/1 	0/0 	1/0 	0/0 	0/1 	0/0
+    1	    865627	.	G	A	.	    .	    EFF=MIS	GT	    0/0	    1/0 	0/1 	0/0 	0/0 	1/0 	0/1 	0/0 	0/0 	1/1 	0/1 	1/0
+    1	    865664	.	G	A	.	    .	    EFF=SYN	GT	    0/1	    0/0 	0/1 	0/0 	0/1 	0/0 	0/0 	0/1 	0/1 	0/0 	0/1 	0/0
+    1	    865691	.	C	T	.	    .	    EFF=MIS	GT	    1/0	    1/0 	0/1 	0/0 	1/0 	1/0 	0/1 	0/0 	1/0 	1/1 	0/1 	0/1
+    """) # noqa
+
+    return vcf_path
+
+
+@pytest.fixture
+def multivcf_split2_vcf(tmp_path_factory: pytest.TempPathFactory) -> Path:
+    root_path = tmp_path_factory.mktemp("mutlivcf_split1")
+    vcf_path = setup_vcf(root_path / "vcf_data" / "in.vcf", """
+    ##fileformat=VCFv4.2
+    ##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+    ##INFO=<ID=EFF,Number=1,Type=String,Description="Effect">
+    ##contig=<ID=1>
+    #CHROM	POS	    ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	f4.mom	f4.dad	f4.p1	f4.s1	f5.mom	f5.dad	f5.p1	f5.s1	f3.s1
+    1	    865582	.	C	T	.   	.   	EFF=SYN	GT  	1/1 	0/0 	0/1 	0/1 	1/1 	0/0 	1/0 	0/1 	./.
+    1	    865583	.	G	A	.   	.   	EFF=SYN	GT  	0/0 	1/1 	0/1 	0/1 	0/0 	1/1 	1/0 	0/1 	./.
+    1	    865624	.	G	A	.   	.   	EFF=MIS	GT  	1/0 	0/0 	1/0 	0/0 	1/0 	0/0 	1/0 	0/0 	./.
+    1	    865627	.	G	A	.   	.   	EFF=MIS	GT  	0/0 	1/1 	1/0 	0/0 	0/0 	1/1 	1/0 	1/0 	./.
+    1	    865664	.	G	A	.   	.   	EFF=SYN	GT  	0/1 	0/0 	0/1 	0/0 	0/1 	0/0 	0/0 	0/1 	./.
+    1	    865691	.	C	T	.   	.   	EFF=MIS	GT  	1/0 	1/1 	1/0 	1/0 	1/0 	1/1 	0/1 	0/1 	./.
```

Comment:
Chromosome names and positions do not correspond to `acgt_gpf` reference genome.